pkg/cmd/cli/secrets/addsecret_to_obj.go

```diff
@@ -0,0 +1,268 @@
+package secrets
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"strings"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api/meta"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/client"
+	cmdutil "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/util"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/resource"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/runtime"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	AttachSecretRecommendedName = "add"
```

Comment:
Can you update the constants to reflect the name?

After reviewing this, I forgot that the command was 'add' and not 'attach'.  I almost commented that I don't like the verb 'attach'.